src/features/SwapAssets/components/SwapAssets.tsx

```diff
@@ -321,11 +324,8 @@ const SwapAssets = (props: Props) => {
 };
 
 const mapStateToProps = (state: AppState) => {
-  const network = selectDefaultNetwork(state) as Network;
-
   return {
     isDemoMode: getIsDemoMode(state),
-    baseAsset: getBaseAssetByNetwork(network)(state) as ExtendedAsset,
     settings: getSettings(state)
   };
```

Comment:
Why does some stuff use `mapStateToProps`, and other stuff `useSelector`? Maybe just move it all to hooks.